package.json

```diff
@@ -110,6 +111,7 @@
     "webpack-cli": "^4.9.1"
   },
   "lint-staged": {
-    "*.{js,ts,jsx,tsx,json,md,css,scss,yml,yaml}": "prettier --write"
+    "*.{js,ts,jsx,tsx,json,md,css,scss,yml,yaml}": "prettier --write",
+    "schema.prisma": "prisma format"
```

Comment:
TIL you can configure the gits hook path which is what husky does... Was confused as all my .git/hooks files have the .sample extension :dizzy_face:

Reply:
Oh, I've never even looked in my `.git`-folders 😅